### PR TITLE
Preping for Production release

### PR DESCRIPTION
**Pull request by yomiodeleye**
### What
- 301 check spaces at end of translations
- 308 check full stop
- release.tmpl - also in other partials check alt text of kitemark, naming and captials
- comment for defaultWindowSize
- mapper.go line 348 consider returning default to avoid panic

### Who can review
FD

### CODE REVIEW

**Comment by gilesburdett** [comment]
Welsh locale has trailing spaces on lines:
- 301
- 313
- 317
- 321

**Review by gilesburdett — CHANGES_REQUESTED** [review]

File: assets/templates/release.tmpl [hunk 1]
```diff
@@ -24,7 +24,7 @@
       >
         <img
           src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/uksa-kitemark.svg"
-          alt="UK Statistics Authority kitemark"
+          alt="UK Statistics Authority Kitemark"
```
[the remark this answers is not in the document]
Comment (yomiodeleye) [reply assets/templates/release.tmpl:27]: I think the comment wast talking about capitalising 'Kitemark'

**Comment by yomiodeleye** [comment]
regarding https://github.com/ONSdigital/dp-frontend-release-calendar/pull/106#issuecomment-1234277285
if we remove the spacing, the rendering in the frontend will be affected.  Because the template engine also factoring in the space from translations too.

File: mapper/mapper.go [hunk 2]
```diff
@@ -306,6 +306,7 @@ func CreateReleaseCalendar(basePage coreModel.Page, params queryparams.Validated
 	return calendar
 }
 
+/// defaultWindowSize is referring to pagination active link. Any links to the left or right of activeLink should not be grater than 5
```
Comment (gilesburdett) [inline mapper/mapper.go:309]: Please replace this with:
```
// The current page number sits within a window, and the window size determines the 
// number of pages around the current page. For example a window size of 3 with the
// current page shown in () would give: 
// - (1) 2 3 at the start of the page range
// - 8 9 (10) at the end of the page range
// - 1 ... 5 (6) 7 ... 10 in the middle of the page range
```

**Review by gilesburdett — CHANGES_REQUESTED** [review]
Please change the comment to the text provided

**Review by gilesburdett — APPROVED** [review]

**Pull request merged**